test/integration/cypress/integration/BenchmarkPage.spec.ts

```diff
@@ -35,4 +36,10 @@ context("Benchmark page", () => {
         });
     });
   });
+
+  it("should link back to benchmarks page", () => {
```

Comment:
This will go away and we'll put the breadcrumb testing at the deepest level of the benchmark page hierarchy.